xml/System.Net.Sockets/Socket.xml

```diff
@@ -242,18 +242,18 @@ This method populates the <xref:System.Net.Sockets.Socket> instance with data ga
       <Docs>
         <param name="socketType">One of the <see cref="T:System.Net.Sockets.SocketType" /> values.</param>
         <param name="protocolType">One of the <see cref="T:System.Net.Sockets.ProtocolType" /> values.</param>
-        <summary>Initializes a new instance of the <see cref="T:System.Net.Sockets.Socket" /> class using the specified socket type and protocol.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.Sockets.Socket" /> class using the specified socket type and protocol. This constructor is designed to prefer dual-stack sockets.</summary>
```

Comment:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Net.Sockets.Socket" /> class using the specified socket type and protocol. If the operating system supports IPv6, this constructor will create an dual-mode socket; otherwise an IPv4 socket will be created.</summary>
```